jobboard/migrations/0002_add_fixed_data.py

```diff
@@ -0,0 +1,42 @@
+from django.db import migrations, transaction
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('jobboard', '0001_initial'),
+    ]
+
+    def generate_data(apps, schema_editor):
+        from jobboard.models import Region, JobTitleKeyword, City
+
+        region_names = [
+            "Central",
+            "South",
+            "North",
+            "Tel Aviv",
+            "Haifa",
+            "Jerusalem"
+        ]
+
+        keywords = [
+            "software",
+            "engineer",
+            "product",
+            "automation"
+        ]
+
+        with transaction.atomic():
+            for name in region_names:
+                Region(name=name).save()
+
+            City(name="Tel Aviv-Yafo", region=Region.objects.get(name="Tel Aviv")).save()
+            City(name="Beersheba", region=Region.objects.get(name="South")).save()
+            City(name="Rishon Leziyyon", region=Region.objects.get(name="Central")).save()
+            City(name="Herziliyya", region=Region.objects.get(name="Tel Aviv")).save()
+
+            for word in keywords:
+                JobTitleKeyword(keyword=word).save()
```

Comment:
What about some example Job records?

Reply:
We were planning to add example jobs when developing tests for functions on the models (we have actually started doing that locally).
We have started adding data to the cities and regions models because that data is real and won't be deleted as we progress.

Reply:
you can add "non-real" data too - when you actually go to production you can add configuration options to you application to know that it is running in production mode and should avoid creating the test data

Reply:
Understood. I added some example job data as instructed.